receiver/splunkhecreceiver/splunk_to_logdata_test.go

```diff
@@ -123,6 +126,24 @@ func Test_SplunkHecToLogData(t *testing.T) {
 			}(),
 			wantErr: nil,
 		},
+		{
+			name: "nil_timestamp",
+			event: splunk.Event{
+				Time:       new(float64),
+				Host:       "localhost",
+				Source:     "mysource",
+				SourceType: "mysourcetype",
+				Index:      "myindex",
+				Event:      "value",
+				Fields: map[string]interface{}{
+					"foo": "bar",
+				},
+			},
+			output: func() pdata.ResourceLogsSlice {
+				return createLogsSlice("value", 0)
```

Comment:
Timestamp is set to 0 on the log record, as mentioned in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1157/files#r511671475